### PR TITLE
add flask-graphql

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@adbeda @agcopenhaver do you both agree to be maintainers?

**Comment by agcopenhaver** [comment]
I agree

**Comment by adbeda** [comment]
Yes

File: recipes/flask-graphql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - flask >=0.7.0
+    - graphql-core >=2.1
+    - graphql-server-core >=1.1
+    - python
+
+test:
+  imports:
+    - flask_graphql
+  requires:
```
Comment (chrisburr) [inline recipes/flask-graphql/meta.yaml]: It looks like you're not using this?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**